website/src/services/welcomeEmailContent.ts

```diff
@@ -0,0 +1,120 @@
+export const welcomeEmailText = `
```

Comment:
**nit-pick:** Maybe add i18n also for welcome email ? If the website is in french, I expect the welcome mail to be in french as well.